src/Bicep.LangServer/Completions/ModuleReferenceCompletionProvider.cs

```diff
@@ -114,7 +115,7 @@ private IEnumerable<CompletionItem> GetTopLevelCompletions(BicepCompletionContex
             }
 
             // Top-level Bicep registry completions
-            AddCompletionItem("br:", null, "Bicep registry", ModuleCompletionPriority.FullPath, "module registry completion");
+            AddCompletionItem(OciArtifactReferenceFacts.SchemeWithColon, null, "Bicep registry", ModuleCompletionPriority.FullPath, "module registry completion");
             if (bicepModuleAliases.Any())
             {
```

Comment:
Prefer an 'IsEmpty' check rather than using 'Any()', both for clarity and for performance
```suggestion
            if (!bicepModuleAliases.IsEmpty)
```

Reply:
9da6b5d9db1e1ca6fde8a55b3300337d08c6d1e0